runtime/src/main/java/org/corfudb/util/serializer/Serializers.java

```diff
@@ -16,11 +19,11 @@
 public class Serializers {
 
     public static final int SYSTEM_SERIALIZERS_COUNT = 10;
-    public static final ISerializer CORFU = new CorfuSerializer((byte) 0);
-    public static final ISerializer JAVA = new JavaSerializer((byte) 1);
-    public static final ISerializer JSON = new JsonSerializer((byte) 2);
-    public static final ISerializer PRIMITIVE = new PrimitiveSerializer((byte) 3);
-    public static final ISerializer QUEUE_SERIALIZER = new CorfuQueueSerializer((byte) 4);
+    public static final CorfuSerializer CORFU = new CorfuSerializer((byte) 0);
```

Comment:
any reason for changing these to the implementation instead of interfaces?

Reply:
In this case changing to implementations helps to reduce the amount of down castings which we are doing anyway, so, since we loose nothing, because you always can use ISerializer anyway, also we make code a bit cleaner